source/grid/tria.cc

```diff
@@ -2092,6 +2092,12 @@ namespace internal
                                            tria_level.refine_flags.size(),
                                          /*RefinementCase::no_refinement=*/0);
 
+          tria_level.refine_choice.reserve(total_cells);
+          tria_level.refine_choice.insert(
```

Comment:
I guess we only need this if the mesh contains tets?

Reply:
Strictly seen yes, yet I think for the changes needed for `MGTwoLevelTransfer::reinit_geometric_transfer()` it will be easier if it is available for all meshes.